recipes/aeron/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.6.1)
+project(test_package)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
```

Comment:
```suggestion
```

CMake verbose makefile is not allowed. I forgot to remove it.

Reply:
Yeah, thanks.